### PR TITLE
Bugfix/download

### PR DESCRIPTION
**Pull request by jcorona48**
### Issue Info

When you click on any of the buttons to download, it sends you to a blank page that downloads. When you accept, it returns to the main page and downloads again. By duplicating the downloads, to solve this it is proposed to eliminate the jump to a blank page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,46 @@ Create commits & pull requests easily.  `Cominnek` is based on [TAYO](https://do
...
+sudo dpkg -i ./cominnek-4.0.0.deb
+```
+4. Verify Installation
+```bash
+cominnek -v
+```
+
+### Install dependencies for auth.
+If you have problem with xdg-open follow this steps
+
```
Comment (isaacismaelx14) [inline README.md:65]: Why are these changes here?

Comment (jcorona48) [reply README.md:65]: Idk, i will see that
